taiga.js.org

OfficialCRUGG opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

I named my Subdomain taiga.js.org instead of taigadiscord.js.org as the discord is just there because we are a Discord Bot and the username Taiga is used by a user who isn't actively using GitHub at the moment. The [wiki page](https://github.com/js-org/js.org/wiki/Subdomain-Determination) wasn't too exact, so I hope this is okay. taigadiscord.js.org would be okay too, if taiga.js.org is not possible.

MattIPv4 commented on cnames_active.js:
```diff
@@ -661,7 +661,7 @@ var cnames_active = {
   "g": "nodebox.github.io/g.js",
   "gal": "galmail.github.io", // noCF? (don´t add this in a new PR)
-  "gamedevcontestal": "fromdenisvieira.github.io/gamedevcontestal", // noCF? (don´t add this in a new PR)
+  "gamedevconal": "fromdenisvieira.github.io/gamedevcontestal", // noCF? (don´t add this in a new PR)
```
Why are you modifying this entry?

MattIPv4 commented on cnames_active.js:
```diff
@@ -1755,6 +1755,7 @@ var cnames_active = {
   "syr": "dmikey.github.io/syr",
   "tagster": "goschevski.github.io/tagster", // noCF? (don´t add this in a new PR)
   "taha": "orcxzjeeeee.github.io/taha",
+  "taiga": "taigadiscord.github.io"
```
Missing comma at the end of your request line.

OfficialCRUGG commented:
Fixed both issues. The other one I changed was an accident. I used to search feature, presses enter and then somehow accidently added a line break to that and somehow removed part of it. My mistake.

indus commented:
Welcome https://taiga.js.org
